README.md

```diff
@@ -1,77 +1,119 @@
+
+# Note the file just uploaded
+In [7]: client.get_files(4)
+Out[7]:
+[File(id=6, name='MFVA', version='1.0', version_code='6'),
+ File(id=7, name='MFVA', version='1.0', version_code='6')]
+
+# Get analyses for the new file
+In [8]: client.get_analyses(6)[:3]
+Out[8]:
+[Analysis(id=267, risk=2, status=3, cvss_base=6.8,
+          findings=[{'title': None, 'description': 'Unprotected service: com.appknox.mfva.ExportedService'}],
+          updated_on='2017-06-27 08:28:35.166608+00:00', vulnerability_id=1),
+ Analysis(id=235, risk=3, status=3, cvss_base=7.3,
+          findings=[{'title': None, 'description': 'pathPrefix=/'}],
+          updated_on='2017-06-27 08:28:35.240543+00:00', vulnerability_id=2),
+ Analysis(id=236, risk=3, status=3, cvss_base=7.7,
+          findings=[{'title': None, 'description': 'Debug enabled within the app'}],
+          updated_on='2017-06-27 08:28:35.296126+00:00', vulnerability_id=3)]
+
+# Note the vulnerability_id for Analysis(id=235). To get details about the vulnerability
+In [9]: client.get_vulnerability(2)
+Out[9]: Vulnerability(name='Improper Content Provider Permissions',
+                      description='A content provider permission was set to allow access from any other app on the device.
+                                   Content providers may contain sensitive information about an app and therefore should not be shared.',
+                      intro="The `ContentProvider` class provides a mechanism for managing and sharing data with other applications.
+                             When sharing a provider's data with other apps, access control should be carefully implemented to prohibit unauthorized access to sensitive data.")
 ```
 
-### Uploading your App
+## Command-line interface
 
 ```
-appknox upload /path/to/your/android.apk
-appknox upload /path/to/your/ios.ipa
+$ appknox login -h https://api.appknox.com
+Username: viren
+Password:
+Logged in to https://api.appknox.com
+
+$ appknox projects
+  id  created_on             file_count  package_name                     platform  updated_on
+----  -------------------  ------------  -----------------------------  ----------  -------------------
+   3  2017-06-23 07:19:26             3  org.owasp.goatdroid.fourgoats           0  2017-06-23 07:26:55
+   4  2017-06-27 08:27:54             2  com.appknox.mfva                        0  2017-06-27 08:30:04
+
+$ appknox files 4
+  id  name      version    version_code
+----  ------  ---------  --------------
+   6  MFVA            1               6
+   7  MFVA            1               6
 ```
 
-### Help
+## Usage
+
 ```
-appknox --help
+Usage: appknox [OPTIONS] COMMAND [ARGS]...
+
+  Command line wrapper for the Appknox API
+
+Options:
+  -v, --verbose  Specify log verbosity.
+  --help         Show this message and exit.
+
+Commands:
+  analyses       List analyses for file
+  dynamic_start  Start dynamic scan for file
+  dynamic_stop   Stop dynamic scan for file
```

Comment:
true, subcommands would have made more sense, but click doesn't have a straightforward way to implement nested subcommands